aea/skills/base.py

```diff
@@ -510,6 +510,14 @@ def parse_module(  # pylint: disable=arguments-differ
         """
         return _parse_module(path, handler_configs, skill_context, Handler)
 
+    @property
+    def protocol_dialogues(self) -> "Dialogues":  # type: ignore # noqa: F821
+        """Protocol dialogues"""
+        if self.SUPPORTED_PROTOCOL is None:
+            raise ValueError(f"SUPPORTED_PROTOCOL not set on {self}")
+        attribute = cast(PublicId, self.SUPPORTED_PROTOCOL).name + "_dialogues"
```

Comment:
On consideration, this doesn't make much sense. The attribute is user defined via the skill.yaml.

Reply:
Will keep open but won't merge for now as a result.